### PR TITLE
Correctly format the number in the drawing

### LINKED ISSUE #1178: Draw feature improvements
- [ ] -todo- the length and area should be formatted (34'000 instead of 34000)
 - [x] -todo- the move icon should be on top (above the feature and the label) #1196 
 - [x] -todo- recenter the view to the selected feature (when a feature is selected in the list)

### PR DESCRIPTION
**Pull request by sbrunner**
fix #1178
replace #1197

### CODE REVIEW

**Comment by fredj** [comment]
instead of creating `externs/angular-add.js` you can add the content to `externs/closure-compiler.js`

File: externs/ngeox.js [hunk 1]
```diff
@@ -1114,3 +1114,25 @@ ngeox.BackgroundEvent = function() {};
  * @type {ol.layer.Base}
  */
 ngeox.BackgroundEvent.prototype.previous;
+
+/**
+ * Format a number with a pressision.
```
Comment (fredj) [inline externs/ngeox.js]: precision instead of pressision

File: src/services/filters.js [hunk 2]
```diff
@@ -31,6 +31,150 @@ ngeo.Scalify = function($filter) {
...
 
 /**
+ * A filter used to format a number with a precision, using the locale.
+ *
+ * Arguments:
+ * - opt_precision: The used precision, default is 3.
+ *
+ * Examples:
+ *
+ *      {{0.1234 | ngeoNumbrer}} => 0.123
```
Comment (fredj) [inline src/services/filters.js]: ngeoNumber instead of ngeoNumbrer

File: src/services/filters.js [hunk 3]
```diff
@@ -31,6 +31,150 @@ ngeo.Scalify = function($filter) {
...
+      decimal.length === 0 ? '' : decimalSep + decimal
+    );
+  };
+  return result;
+};
+
+ngeo.module.filter('ngeoNumber', ngeo.Number);
+
+/**
+ * A filter used tormat a number with the prefix and unit
```
Comment (fredj) [inline src/services/filters.js]: to format instead of tormat

**Comment by sbrunner** [comment]
Thanks @fredj everything done :-)

**Comment by fredj** [comment]
LGTM

**Pull request merged**